app/models/item.rb

```diff
@@ -1,6 +1,7 @@
 class Item < ApplicationRecord
 
```

Comment:
validations changed to comply with user story 23
we still need to handle placeholder images?